src/chanjo2/crud/intervals.py

```diff
@@ -58,3 +63,45 @@ def get_genes(db: Session, build: Builds, limit: int) -> List[Gene]:
         .limit(limit)
         .all()
     )
+
+
+def create_db_transcript(db: Session, transcript: TranscriptBase) -> SQLTranscript:
+    """Create a SQL transcript object."""
+
+    ensembl_gene = (
+        db.query(SQLGene)
+        .filter(SQLGene.ensembl_id == transcript.ensembl_gene_id)
+        .first()
+    )
+
+    return SQLTranscript(
+        chromosome=transcript.chromosome,
+        start=transcript.start,
+        stop=transcript.stop,
+        ensembl_id=transcript.ensembl_id,
+        ensembl_gene_ref=ensembl_gene.id if ensembl_gene else None,
+        ensembl_gene_id=transcript.ensembl_gene_id,
+        refseq_mrna=transcript.refseq_mrna,
+        refseq_mrna_pred=transcript.refseq_mrna_pred,
+        refseq_ncrna=transcript.refseq_ncrna,
+        refseq_mane_select=transcript.refseq_mane_select,
+        refseq_mane_plus_clinical=transcript.refseq_mane_plus_clinical,
+        build=transcript.build,
+    )
+
+
+def bulk_insert_transcripts(db: Session, transcript_list: List[TranscriptBase]):
```

Comment:
```suggestion
def bulk_insert_transcripts(db: Session, transcripts: List[TranscriptBase]):
```
We already know it is a list and what type of object from the type hint 😄

Reply:
Fair enough!